### ISSUE TITLE
Sitewide accessibility discovery

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9492]
#### From #9498

As a **Sitewide accessibility lead**, I need an understanding of the accessibility landscape (see this mural as a [rough start at this](https://app.mural.co/t/vagov6717/m/vagov6717/1655383090408/4d4957dbe9fc3a31ce45f280b1188fa8cdf7dc24?sender=kevinwalsh8610))
- at VA
- within OCTO-DE
- within the Sitewide crew

so that i know who's who, and understand the scope of the Sitewide vs other teams with respect to improving accessibility of VA.gov.

As **a member of the OCTO-DE accessibility practice area** (team?), i want to understand the role and responsibilities of the Sitewide Crew accessibility lead, so that i know how to include them in various initiatives. 

#### Via #9493 / #9494  

As **a Sitewide Crew**, I want actionable metrics to help prioritize accessibility initiatives 
- by product
- by source of problem (Editor training/guidance, CMS application, FE templates/CSS, VA Design system, etc)
- modernized / non-modernized
- others TBD 

#### Via #9495 

As an accessibility lead or someone in Sitewide crew working on accessibility issues, i have a prioritized backlog of accessibility issues. 

### What's not included 

- #9443. This will happen either in parallel, or subsequently. 
  
## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [x] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
It has been determined with the PO that the Accessibility Lead's focus should be on the CMS products rather than the FE.

**Comment by EWashb** [comment]
All accessibility discovery in this epic is complete
